apps/native-component-list/src/screens/Blob/expo-blob/BlobPerformanceTestScreen.tsx

```diff
@@ -0,0 +1,267 @@
+    iterations: 100,
+  },
+  {
+    key: 'bmp-file-test',
+    blobOperation: async () => {
+      const asset = Asset.fromModule(
+        require('../../../../assets/expo-blob/performance-test-2mb.bmp')
+      );
+      await asset.downloadAsync();
+      const uri = asset.localUri || asset.uri;
+      const response = await fetch(uri);
+      const blobResponse = await response.blob();
+      const base64 = await blobToBase64(blobResponse);
+      const blob = new Blob([base64]);
+      blob.slice(0, 1000);
+    },
+    expoBlobOperation: async () => {
+      const asset = Asset.fromModule(
+        require('../../../../assets/expo-blob/performance-test-2mb.bmp')
+      );
+      await asset.downloadAsync();
+      const uri = asset.localUri || asset.uri;
+      const response = await fetch(uri);
+      const blobResponse = await response.blob();
+      const base64 = await blobToBase64(blobResponse);
+      const blob = new ExpoBlob([base64]);
+      blob.slice(0, 1000);
+    },
+    title: 'File Test (2MB BMP)',
+    iterations: 5,
+  },
+  {
+    key: 'audio-file-test',
+    blobOperation: async () => {
+      const asset = Asset.fromModule(
+        require('../../../../assets/expo-blob/performance-test-1mb.mp3')
+      );
+      await asset.downloadAsync();
+      const uri = asset.localUri || asset.uri;
+      const response = await fetch(uri);
+      const blobResponse = await response.blob();
+      const base64 = await blobToBase64(blobResponse);
+      const blob = new Blob([base64]);
+      blob.slice(0, 1000);
+    },
+    expoBlobOperation: async () => {
+      const asset = Asset.fromModule(
+        require('../../../../assets/expo-blob/performance-test-1mb.mp3')
+      );
+      await asset.downloadAsync();
+      const uri = asset.localUri || asset.uri;
+      const response = await fetch(uri);
+      const blobResponse = await response.blob();
+      const base64 = await blobToBase64(blobResponse);
+      const blob = new ExpoBlob([base64]);
+      blob.slice(0, 1000);
+    },
+    title: 'File Test (1MB Audio)',
+    iterations: 25,
+  },
+  {
+    key: 'video-file-test',
+    blobOperation: async () => {
+      const asset = Asset.fromModule(
+        require('../../../../assets/expo-blob/performance-test-1mb.mp4')
+      );
+      await asset.downloadAsync();
+      const uri = asset.localUri || asset.uri;
+      const response = await fetch(uri);
+      const blobResponse = await response.blob();
+      const base64 = await blobToBase64(blobResponse);
+      const blob = new Blob([base64]);
+      blob.slice(0, 1000);
+    },
+    expoBlobOperation: async () => {
+      const asset = Asset.fromModule(
+        require('../../../../assets/expo-blob/performance-test-1mb.mp4')
+      );
+      await asset.downloadAsync();
+      const uri = asset.localUri || asset.uri;
+      const response = await fetch(uri);
+      const blobResponse = await response.blob();
+      const base64 = await blobToBase64(blobResponse);
```

Comment:
I don't think we should measure how long this function takes in the test – this skews the results, no?